Cargo.lock

```diff
@@ -81,6 +81,10 @@ dependencies = [
  "generic-array",
 ]
 
+[[package]]
+name = "fsb_rust"
+version = "0.1.0"
```

Comment:
Since you've already changed the crate name, re-generate the lock file, so it will use `fsb` instead of `fsb_rust`? I have shared ownership of the `fsb` crate with the hashes group, so we can publish it immediately after merge.